rust-runtime/aws-smithy-types/src/lib.rs

```diff
@@ -13,455 +13,17 @@
-                kind: TryFromNumberErrorKind::I64ToFloatLossyConversion(..)
-            }
-        ));
-        assert!(matches!(
-            f64::try_from(Number::NegInt(-significand_max_i64 - 1)).unwrap_err(),
-            TryFromNumberError {
-                kind: TryFromNumberErrorKind::I64ToFloatLossyConversion(..)
-            }
-        ));
-    }
-
-    #[test]
-    fn to_f32() {
-        assert_eq!(f32::try_from(Number::PosInt(69u64)).unwrap(), 69f32);
-        assert_eq!(f32::try_from(Number::NegInt(-69i64)).unwrap(), -69f32);
-
-        let significand_max_u64: u64 = 1 << 24;
-        let significand_max_i64: i64 = 1 << 24;
-
-        assert_eq!(
-            f32::try_from(Number::PosInt(significand_max_u64)).unwrap(),
-            16777216f32
-        );
-
-        assert_eq!(
-            f32::try_from(Number::NegInt(significand_max_i64)).unwrap(),
-            16777216f32
-        );
-        assert_eq!(
-            f32::try_from(Number::NegInt(-significand_max_i64)).unwrap(),
-            -16777216f32
-        );
-
-        assert!(matches!(
-            f32::try_from(Number::PosInt(significand_max_u64 + 1)).unwrap_err(),
-            TryFromNumberError {
-                kind: TryFromNumberErrorKind::U64ToFloatLossyConversion(..)
-            }
-        ));
-
-        assert!(matches!(
-            f32::try_from(Number::NegInt(significand_max_i64 + 1)).unwrap_err(),
-            TryFromNumberError {
-                kind: TryFromNumberErrorKind::I64ToFloatLossyConversion(..)
-            }
-        ));
-        assert!(matches!(
-            f32::try_from(Number::NegInt(-significand_max_i64 - 1)).unwrap_err(),
-            TryFromNumberError {
-                kind: TryFromNumberErrorKind::I64ToFloatLossyConversion(..)
-            }
-        ));
-
-        for val in [69f64, f64::NAN, f64::INFINITY, f64::NEG_INFINITY] {
-            assert!(matches!(
-                f32::try_from(Number::Float(val)).unwrap_err(),
-                TryFromNumberError {
-                    kind: TryFromNumberErrorKind::F64ToF32LossyConversion(..)
-                }
-            ));
-        }
-    }
-
-    #[test]
-    fn to_f64_lossy() {
-        assert_eq!(Number::PosInt(69u64).to_f64_lossy(), 69f64);
-        assert_eq!(
-            Number::PosInt((1 << 53) + 1).to_f64_lossy(),
-            9007199254740992f64
-        );
-        assert_eq!(
-            Number::NegInt(-(1 << 53) - 1).to_f64_lossy(),
-            -9007199254740992f64
-        );
-    }
-
-    #[test]
-    fn to_f32_lossy() {
-        assert_eq!(Number::PosInt(69u64).to_f32_lossy(), 69f32);
-        assert_eq!(Number::PosInt((1 << 24) + 1).to_f32_lossy(), 16777216f32);
-        assert_eq!(Number::NegInt(-(1 << 24) - 1).to_f32_lossy(), -16777216f32);
-        assert_eq!(
-            Number::Float(1452089033.7674935).to_f32_lossy(),
-            1452089100f32
-        );
-    }
-}
+pub use blob::Blob;
+pub use document::Document;
+pub use number::Number;
```

Comment:
Looks like the `DateTime` and `Error` re-exports were lost:
```suggestion
pub use blob::Blob;
pub use date_time::DateTime;
pub use document::Document;
pub use error::Error;
pub use number::Number;
```

Reply:
Sorry I didn't notice that.  
It seems like it got lost when I merged the other branch.  

Thank you for letting me know!